src/style-spec/validate/validate_function.js

```diff
@@ -1,11 +1,13 @@
 
 import ValidationError from '../error/validation_error';
 import getType from '../util/get_type';
+import extend from '../util/extend';
```

Comment:
I don't think you need `extend` anymore 🙃

Reply:
yepp